app/models/node.rb

```diff
@@ -1070,7 +1070,7 @@ def draft_url(base_url)
     base_url + '/notes/show/' + nid.to_s + '/' + token
   end
 
-  def fetch_comments(user)
+  def comments_to_be_read(user)
```

Comment:
My thinking was `comments_viewable_by(user) ` was a more descriptive method name. What do you think? Thanks again

Reply:
Yes your rit @cesswairimu .......

Reply:
cool, ping me if you need any help. Thanks